code/modules/admin/admin.dm

```diff
@@ -96,8 +95,9 @@ var/global/floorIsLava = 0
 	var/list/dat = list()
 
 	var/p_age
+	var/key_c = ckey(key)
 	for(var/client/C in GLOB.clients)
-		if(C.ckey == key)
+		if(C.ckey == ckey(key_c))
```

Comment:
```suggestion
		if(C.ckey == ckey(key))
```

Reply:
Calling a string transformation for every client on the entire server seems slow unless byond compiler optimizes this out which i doubt